### PR TITLE
Improve handling of movements in 3rd dimension

### LINKED ISSUE #5233: Toggling between two slices does not visibly move other viewports
## Context
Viewing a dataset, toggling between two adjacent slices (either by alternating d/f or mousewheel), zoomed in to mag1

## Expected Behavior
The yz, xz viewports and 3d viewport should show in which direction the move was taking place

## Current Behavior
Those viewports do not visibly change

## Steps to Reproduce the bug
Viewing a dataset, toggling between two adjacent slices (either by alternating d/f or mousewheel), zoomed in to mag1

## Your Environment for bug
Firefox 86 Windows 10

## Note
https://github.com/scalableminds/webknossos/pull/5201 probably causes this. Philipp already explained that to solve this while keeping the desired behavior from there, the code would have to differentiate between holding keys pressed and hitting them singly. Let’s see if other users complain about this. If it’s just me, the priority is not high.

### PR DESCRIPTION
**Pull request by philippotto**
This PR removes the old "clamp to edge" behavior on initial third-dimension movements, since this had undesired side effects (such as #5233). As a replacement, the keyboard delay is adapted according to the current position. See code comments for more details.

### URL of deployed dev instance (used for testing):
https://fixvisiblemove.webknossos.xyz

### Steps to test:
- open a dataset (ideally with a real-world scale, such as 11,11,24)
- turn down the move value to something unbearably slow (e.g., 5 nm/s which I allowed temporarily for this PR)
- play around with the position field (which now supports floats in this PR <-- I will undo this)
  - set the z-value to x.99 and move forward with f (keep the key pressed)
  - set the z-value to x.00 and move forward with f (keep the key pressed)
  - the experienced keyboard delay for the first scenario should be higher than for the second. you can also check the console output to see the calculated delays.

### Issues:
- fixes #5233 
- hopefully softens #5653 

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
@fm3 Feel free to check out https://fixvisiblemove.webknossos.xyz to see whether it's an improvement with regard to https://github.com/scalableminds/webknossos/issues/5233.

**Comment by fm3** [comment]
Thanks! Yes, my use case is handled better now :)

File: frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js [hunk 1]
```diff
@@ -172,31 +172,27 @@ function createDelayAwareMoveHandler(multiplier: number) {
     const state = Store.getState();
     let direction = Math.sign(multiplier);
 
-    const moveDistanceIn1Second =
-      state.userConfiguration.moveValue / getBaseVoxel(state.dataset.dataSource.scale);
     const { activeViewport } = state.viewModeData.plane;
+    const thirdDim = dimensions.thirdDimensionForPlane(activeViewport);
+    const voxelPerSecond =
+      state.userConfiguration.moveValue / state.dataset.dataSource.scale[thirdDim];
```
Comment (philippotto) [inline frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js:178]: @daniel-wer Using `getBaseVoxelFactors` didn't produce the correct result (the timing was still off). It turned out that it was even easier: just use the nm-based moveValue and divide it by the correct component of the dataset scale. Now, the delay should be right :)

**Review by daniel-wer — APPROVED** [review]
Very nice, feels exactly right now :)

LGTM once the temporary changes are reversed :+1:

**Pull request merged**